WalletWasabi.Gui/Helpers/NotificationHelpers.cs

```diff
@@ -15,39 +17,40 @@ public static INotificationManager GetNotificationManager()
 			return Locator.Current.GetService<INotificationManager>() ?? NullNotificationManager;
 		}
 
-		public static void Notify(string message, string title, NotificationType type, Action onClick = null)
+		public static void Notify(string message, string title, NotificationType type, Action onClick = null, object sender = null)
 		{
+			if (sender is Wallet wallet)
```

Comment:
This is an anti-OOP pattern. If a method has to behave differently depending on the caller then something is not okay. I cannot find a way to make improve  without changing lots of thing so, I think the best will be to go with this as it is and continue thinking.

Reply:
My idea here was to handle and format in one place if the caller wants to indicate the sender of the notification somehow.
The caller is not the sender here.